src/fiatExchanges/CoinbasePaymentSection.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+
+import { useTranslation } from 'react-i18next'
+import { Image, StyleSheet, Text, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import { coinbasePayEnabledSelector } from 'src/app/selectors'
+import Touchable from 'src/components/Touchable'
+import { FetchProvidersOutput } from 'src/fiatExchanges/utils'
+import colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+
+export interface CoinbasePaymentSectionProps {
+  coinbaseProvider: FetchProvidersOutput | undefined
+  setNoPaymentMethods: React.Dispatch<React.SetStateAction<boolean>>
+}
+
+export function CoinbasePaymentSection({
+  coinbaseProvider,
+  setNoPaymentMethods,
+}: CoinbasePaymentSectionProps) {
+  const { t } = useTranslation()
+  const coinbasePayEnabled = useSelector(coinbasePayEnabledSelector)
+
+  if (!coinbaseProvider || coinbaseProvider.restricted || !coinbasePayEnabled) {
+    setNoPaymentMethods(true)
```

Comment:
Since coinbase pay isn't available for cash outs this setNoPaymentMethods will cause the little blurb at the bottom saying 'some payment methods are unavailable' to always show up. I don't think we actually want this experience so i think setNoPaymentMethods can be removed from the component

Reply:
seems like we're likely to continue to running into this issue, might be worth thinking about a more general solution (besides just not taking coinbase pay into account)

Reply:
for example, I'm really not sure we offer bank account cash-outs anywhere

Reply:
Do we want the 'some payment methods are unavailable' text to show up on _cash in_ if CB Pay isn't available?